js/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Prerequisites
 
-- Node.JS 4,5, or 6+
```

Comment:
I actually think that is good to support Node.js 4 since it is the current LTS. See https://github.com/hashrocket/websocket-shootout/pull/37.